library/src/main/res/layout/exomedia_exo_view_layout.xml

```diff
@@ -2,9 +2,11 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
                 xmlns:tools="http://schemas.android.com/tools"
                 android:layout_width="match_parent"
-              android:layout_height="match_parent">
+              android:layout_height="match_parent"
+                android:background="@android:color/black"
```

Comment:
We shouldn't be specifying this color.  That should be left up to the user of the library on the  EMVideoView itself.